src/app/api/googleDrive/googleDrive.api.ts

```diff
@@ -167,4 +167,34 @@ export const getFileContentBase64FromGoogleDrive = async (fileUrl: string) => {
     console.error("Error retrieving file from Google Drive:", error);
     throw error; // Rethrow the error for further handling
   }
+};
+
+export const uploadPDFDocument = async (file: File, body: any) => {
+  const drive = google.drive({ version: "v3", auth });
+  try {
+    if (file.type !== "application/pdf") {
+      throw new Error("Only PDF files are allowed");
+    }
```

Comment:
I didnt tell you this.

We shall allow to upload images aswell. But if is an image we need to convert in PDF

Normally they will take a picture and upload from the phone

Then can continue the process